.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: govuk-content-schemas
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - run: bundle exec rake
```

Comment:
I feel like we're going to hit a consistency issue soon with some apps that use rake default to only run tests and others that use it to run all build tasks including linting. 

As an example Content Publisher does everything in Rake default intentionally: https://github.com/alphagov/content-publisher/blob/c6e81e4a71b9b4a9710638988a6ce0a0366743f9/Rakefile#L10

We're going to have to decide whether we want a simple action file that just calls rake but fails fast or if we want each step called out in steps so that we can see whether lint passed but tests failed (for instance).

Reply:
I'd be in favour of all the steps being separate, for fast failures. I did a bunch of work last year to remove the `lint` Rake tasks as Jenkins itself ran `RuboCop` for us - but I'd be in favour of running each step individually for fewer abstractions. I'll think about this.

Reply:
+1 for `rake`, as then it's consistent with local development. Also less work than having to manually specify all the steps.

Reply:
@issyl0 would it actually lead to faster failures? If `rake` is running all the same steps, then it should fail at the same point; arguably, it's faster because there's less startup overhead. Having been annoyed by Jenkins doing something 'different' to running `rake` locally, I'm very keen on us standardising on just `rake`.

Reply:
If, across all the apps, we can standardise on `rake` running the tests AND RuboCop, that's be good on the whole. But I would like to take into account [Michael's RFC comment](https://github.com/alphagov/govuk-rfcs/pull/123#issuecomment-617175480) about individualised steps being easier to see what failed - but maybe that's for the future.